src/pages/User/Learning/TodaysWord/TodaysWord.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import fetchWithToken from 'API/api';
+import * as S from './StyleTodaysWord';
+import { Button } from 'components/Shared/Buttons/Button';
+import NotificationInformation from 'components/Shared/Atoms/NotificationInformation';
+import ManagedIcon from 'assets/icon/helpers/ManagedIcon';
+import { GlobalContext } from 'utils/GlobalContext';
+
+const TodayWord = () => {
+  const { isLoginUser } = useContext(GlobalContext);
+  const [baseWord, setBaseWord] = useState(null);
+  const [transWord, setTransWord] = useState(null);
+  const [loading, setLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+  const [isShowWord, setIsShowWord] = useState(false);
+  const [isSpeaking, setIsSpeaking] = useState(false);
+  const [idWord, setIdWord] = useState(null);
+
+  const navigate = useNavigate();
+  const handleAddWordClick = () => {
```

Comment:
jak masz takie funkcje jednolinijkowe to może to skrócić: 
`const handleAddWordClick = () => navigate('/settings/add_words');`
 poniżej to samo. 
 Ale to tylko kosmetyczna zmiana